app/config/services/admin.xml

```diff
@@ -609,6 +609,19 @@
             </call>
         </service>
 
+        <!--
+        ######################################################
+        # Biography
+        ######################################################
+        -->
+        <service id="app.admin.biography_executive_office_member" class="AppBundle\Admin\ExecutiveOfficeMemberAdmin">
```

Comment:
can we use the class as id ?

Reply:
Sonata needs this syntax for the security IMO.